pkg/mutation/path/token/scanner_test.go

```diff
@@ -81,26 +81,110 @@ func TestScanner(t *testing.T) {
 				{Type: EOF, Literal: ""},
 			},
 		},
+		{
+			input: `"one"two`,
+			expected: []Token{
+				{Type: IDENT, Literal: "one"},
+				{Type: IDENT, Literal: "two"},
+				{Type: EOF, Literal: ""},
+			},
+		},
+		// Integer token support
+		{
+			input: `0123`,
```

Comment:
have we covered the `abc123` use case? I assume that's all a single identifier?

Reply:
Yep - I added explicit test cases for it now 👍🏻